packages/doc/content/components/components/button/index.mdx

```diff
@@ -3,4 +3,5 @@ title: 'Button'
 metaTitle: 'Buttons'
 metaDescription: 'This is the list of Buttons'
 linkable: false
+opened: false
```

Comment:
Since the doc menu uses an accordion, what you think of the 'collapsed' nomenclature instead of 'opened'.
On the navigation component, we use 'collapsed' (https://github.com/Gympass/yoga/blob/bebb09b5f47b7946668f14428bc5f300a3192475/packages/doc/src/components/Navigation/Navigation.jsx#L139) 

 It's just a detail though, do you think that makes sense?

Reply:
Nice, already done!